src/main/java/de/hpi/datastreams/apps/App.java

```diff
@@ -18,6 +20,10 @@
     public final static String GRADIENTS_TOPIC = "GRADIENTS_TOPIC";
     public final static String WEIGHTS_TOPIC = "WEIGHTS_TOPIC";
 
+    public final static String CLASSIFICATION_DATA_TOPIC = "CLASSIFICATION_DATA_TOPIC";
+    public final static String CLASSIFICATION_OUTPUT_TOPIC = "CLASSIFICATION_OUTPUT_TOPIC";
```

Comment:
Please think of using `PREDICTION` instead of `CLASSIFICATION` as we aim to support regression tasks too.